tex/artikel/Professoren_Vorstellung.tex

```diff
@@ -1,52 +1,53 @@
 % XXX Jedes Jahr Professoren-Texte aktualisieren!
 \section[Eure Profs stellen sich vor]{Eure Professoren stellen sich vor}
-\textbf{Auf den folgenden zwei Seiten stellen sich eure beiden Professoren vor.
-    Sie werden gemeinsam die "Physik~1" bis "Physik~3" lesen.
-    Prof.\ Thiele sowie Prof.\ Wittkowski werden sich dabei um die theoretischen und Prof.\ Fallnich um die experimentellen Aspekte des Studiums kümmern.
-    Zudem stellt sich Prof.\ Wulkenhaar vor, der die Vorlesungen "Mathematik für Studierende der Physik" halten wird (ebenfalls über drei Semester).
-	Da diese drei Professoren euch eine Zeit lang begleiten werden, ist es durchaus interessant zu wissen, was sie gemacht haben, bevor sie an die Uni Münster kamen, und wie ihre aktuelle Forschung aussieht.}
-
- \begin{figure}[b] % [b] option places the figure at the bottom
-	\centering
-\includegraphics[width=0.8\columnwidth, height=0.25\textheight]{res/vorstellungsfotos/profs_ws23.jpg}
-	\caption*{\centering Gruppenfoto der Physik~1-3 Dozenten; in Ihrem Bezugssystem von links nach rechts: Carsten~Fallnich, Raphael~Wittkowski, Karol~Kovarik und Uwe~Thiele.}
-\end{figure}
+\textbf{Auf den folgenden zwei Seiten stellen sich die Professoren vor, die Euch im integrierten Kurs Physik 1-3 betreuen werden. 
+Die Theoretische Physik wird in diesem Kurs von Prof.\ Dr.\ Michael Klasen, die Experimentelle Physik von Prof.\ Dr.\ Martin Salinga vertreten. 
+Die zugehörigen Übungen und Tutorien sowie das sehr nützliche Mathematik-Repetitorium werden von PD Dr.\ Karol Kovarik betreut. 
+Prof.\ Dr.\ Lutz Hille wird in den ersten drei Semestern die Vorlesungen "Mathematik für Studierende der Physik" halten. Da Euch diese Professoren eine Zeit lang begleiten werden, ist es durchaus interessant zu wissen, was sie gemacht haben, bevor sie an die Universität Münster kamen, und womit sie sich in ihrer aktuellen Forschung beschäftigen.}
```

Comment:
```suggestion
\textbf{Auf den folgenden zwei Seiten stellen sich die Professoren vor, die euch im integrierten Kurs Physik 1-3 betreuen werden. 
Die Theoretische Physik wird in diesem Kurs von Prof.\ Dr.\ Michael Klasen, die Experimentelle Physik von Prof.\ Dr.\ Martin Salinga vertreten. 
Die zugehörigen Übungen und Tutorien sowie das sehr nützliche Mathematik-Repetitorium werden von PD Dr.\ Karol Kovarik betreut. 
Prof.\ Dr.\ Lutz Hille wird in den ersten drei Semestern die Vorlesungen "Mathematik für Studierende der Physik" halten. Da euch diese Professoren eine Zeit lang begleiten werden, ist es durchaus interessant zu wissen, was sie gemacht haben, bevor sie an die Universität Münster kamen, und womit sie sich in ihrer aktuellen Forschung beschäftigen.}
```

"Euch" ist groß geschrieben, was ziemlich veraltet ist. Vielleicht ist das ein Versehen?

Reply:
Ne das ist tatsächlich noch so in Gebrauch, so ähnlich wie man das 'Dir' zur Betonung auch groß schreibt. 
Das passt also alles so.

Reply:
Zitat von https://www.duden.de/sprachwissen/sprachratgeber/Gross-oder-Kleinschreibung-von-duDu-und-ihrIhr :

> In der Regel werden die Formen des Anredepronomens du kleingeschrieben, ebenso dann auch die der Mehrzahl ihr. Die Kleinschreibung gilt darüber hinaus auch für die Formen der Possessivpronomen dein, euer und auch für deinesgleichen, deinerseits usw.

Reply:
In derselben Quelle steht aber auch: "Allgemein gilt: In allen Textformen, in denen der Autor selbst den Leser persönlich anredet, kann man du/Du und die genannten Wörter großschreiben. Das gilt dann zum Beispiel für Zettelmitteilungen in WGs (Katinka, du/Du wirst durchaus nicht gedisst, wenn du/Du auch mal den Müll runterbringst), Bemerkungen von Lehrern in den Heften ihrer Schüler (Da hattest du/Du wirklich einen ganz tollen Einfall, Steffi) und, und, und. In all diesen Fällen empfehlen wir übrigens die Großschreibung." ;) 

Es ist also nicht falsch (vlt ungewohnt). 
Das wäre daher glaube ich Erbsenzählerei.